Gemfile.lock

```diff
@@ -86,7 +86,7 @@ GEM
       activerecord (>= 5.0, < 6.2)
```

Comment:
I'm not sure this file and all the other gems should be here

Reply:
They shouldn't, all fixed ✔️